console/src/main/webapp/manager/app/components/users/users.tpl.html

```diff
@@ -9,10 +9,6 @@
     <h3 ng-if="!users.activeRole" class="roles-title">{{users.users.length}} <span translate>role.users</span></h3>
     <h3 ng-if="users.activeRole" class="roles-title">
       {{users.activeRole.cn}}
-      <button class="btn btn-default btn-xs delete" title="{{ 'role.delete' | translate }}"
-        ng-click="users.deleteRole()" ng-show="users.canDelete">
-        <i class="glyphicon glyphicon-trash"></i>
-      </button>
     </h3>
     <p ng-if="users.activeRole">
       <em class="content-description" contenteditable="true" ng-init="users.initEditable()">{{users.activeRole.description}}</em>
```

Comment:
I guess editable description should removed, too.

Reply:
True, done